internal/errors/error.go

```diff
@@ -67,10 +67,10 @@ type wrapError struct {
 	frame   xerrors.Frame
 }
 
-type myprinter struct {
+type FormatErrorPrinter struct {
```

Comment:
This seemed like a nicer solution than declaring another identical type in `decode.go` for the printer struct, but I don't have to do it this way if that's preferred.